docs/readthedocs/allpages.md

```diff
@@ -0,0 +1,17 @@
+(allpages)=
+
+# All Pages
+
+Here are all available pages listed:
+
+```{toctree}
+---
+maxdepth: 1
+---
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   *
+   !allpages
+```
```

Comment:
This stacked toctree seems to cause the error
```
/home/docs/checkouts/readthedocs.org/user_builds/simona/checkouts/771/docs/readthedocs/allpages.md:7: WARNING: toctree contains reference to nonexisting document '.. toctree::'
/home/docs/checkouts/readthedocs.org/user_builds/simona/checkouts/771/docs/readthedocs/allpages.md:7: WARNING: toctree contains reference to nonexisting document '   :maxdepth: 2'
/home/docs/checkouts/readthedocs.org/user_builds/simona/checkouts/771/docs/readthedocs/allpages.md:7: WARNING: toctree contains reference to nonexisting document '   :glob:'
/home/docs/checkouts/readthedocs.org/user_builds/simona/checkouts/771/docs/readthedocs/allpages.md:7: WARNING: toctree contains reference to nonexisting document '   *'
/home/docs/checkouts/readthedocs.org/user_builds/simona/checkouts/771/docs/readthedocs/allpages.md:7: WARNING: toctree contains reference to nonexisting document '   !allpages'
```

Reply:
Yes, unfortunately the solution doesn't work because you can apparently only have one toctree. @sebastian-peter and I imagined that we would have an index page, a search page and a page where all pages are listed chronologically (Modindex/allpages). Unfortunately I haven't found any other solution apart from toctree. But I'll keep looking now.

Reply:
@pierrepetersmeier I think we can skip the listing of all pages, if it's not easily implementable. Our main navigation should fit the purpose anyways.

Reply:
Just fyi, I found out what the index is actually supposed to be. You can add index entries on all pages with `.. index:: Entry name`, and they're then added to the index linking back to the page that they appear on. So it's a literal index similar to how it works in books. Since we currently don't use this, it's definitely fine to remove it.

Reply:
Alright, good to know, maybe we'll need it sometime.